### ISSUE TITLE
Skybox/Sky textures

### ISSUE THREAD

**Issue by Ganonmaster** [overtools/OWLib#41]
Hi dudes/dudettes,

I'd like to complement you on the wonderful work you've been doing so far.

I've been using the DataTool to extract some maps, but I can't seem to figure out where/how Blizzard stores the Sky textures. Most maps seem to contain fog planes, distant mountains and vistas, and I do see those in the extracted maps, but I can't seem to find the actual sky itself. Am I missing something obvious, or are the sky textures actually missing?

**Comment by ZingBallyhoo** [comment]
Nobody knows how it works yet. If somebody can find one of the textures then I can start to connect the dots.

I have a feeling it's managed by an 00D/04A/08E/08F effect file.

**Comment by healingbrew** [comment]
002 files.

**Comment by ZingBallyhoo** [comment]
Yep, just took a look. The 002 files define a bunch of things ranging from entities to cubemaps and random model looks.

No ETA on when it'll get implemented, probably not for a while.

**Comment by healingbrew** [comment]
I could take a look at it, I've got not much better to do in my spare time right now.

**Comment by healingbrew** [comment]
```
float[12] matrix4x3;  //rst?
float[16] matrix4;
float[16] matrix4;
float[16] matrix4;
guid	  maplogic;   // 003
guid	  texture;    // 004
guid	  lightning;  // 0BD
guid	  shadow;     // 0CB
guid      texture;    // 004
guid      skybox;     // 00C
guid      material;   // 01A
guid	  texture;    // 004
guid	  texture;    // 004
guid	  string;     // 07C
guid	  unknown;    // 0B5
float[16] matrix4;
guid	  texture;    // 004
guid	  texture;    // 004
guid	  texture;    // 004
float[12] matrix4x3;
float[12] matrix4x3;
int[12]   counts;     // ?
```

I'll implement something later this week.

**Comment by healingbrew** [comment]
Implemented in 836f936956e25310776fcdc6cd3a89201ce6c330
